university/src/main/java/nl/quintor/studybits/university/Seeder.java

```diff
@@ -74,64 +58,39 @@ private void seedUniversities() {
         SchemaKey transcriptSchemaKey = universityService.defineSchema("rug", transcriptSchemaDefinition);
         universityService.defineClaim("rug", transcriptSchemaDefinition);
         universityService.addSchema("gent", transcriptSchemaKey);
-    }
-
-
-    private void seedUsers() {
-        University rug = universityService.getUniversity("rug");
-
-        User admin1 = createAdmin("admin1", "Etienne", "Nijboer", "222-11-0001", rug);
-
-        User rugStudent1 = createStudent("student1", "Peter", "Ullrich", "1111-11-0001", rug);
-        addAcademicYears(rugStudent1, "2016/17");
-
-        User rugStudent2 = createStudent("student2", "Margot", "Veren", "1111-11-0002", rug);
-        addAcademicYears(rugStudent2, "2016/17", "2017/18");
 
-        User rugStudent3 = createStudent("student3", "Ko", "de Kraker", "1111-11-0003", rug);
-        addAcademicYears(rugStudent3, "2015/16", "2016/17", "2017/18");
-        addTranscript(rugStudent3, new TranscriptModel("Bachelor of Science, Marketing", "graduated", "2018", "5"));
+        exchangeUniversityClaimIssuerSchemaInfo("gent", "rug");
```

Comment:
Find a better name for this. like `setAcceptedClaimIssuer`

Reply:
That would be a name for the service. In the seeder the issuer SchemaInfo is exchanged, meaning that the issuer DID for all issued claims of Rug are exchanged with Gent. I know its not a great name, but setAcceptedClaimIssuer is not covering it.